app/src/androidTest/java/ch/hikemate/app/ui/profile/ProfileScreenTest.kt

```diff
@@ -100,20 +127,29 @@ class ProfileScreenTest : TestCase() {
                   context.getString(R.string.profile_screen_hiking_level_info_expert)
             }))
 
-    val current = LocalDate.now()
-    val currentDateDayOfMonth = current.dayOfMonth
-    val currentDateMonthText = current.month.toString().lowercase()
+    val current = Calendar.getInstance().time
+    val currentDateDayOfMonthText = SimpleDateFormat("d").format(current).toString()
+    val currentDateMonthText = SimpleDateFormat("MMMM").format(current).lowercase()
```

Comment:
Question: Why have you changed that?

Reply:
Great question! Because of Locale. On my phone, it was written "Novembre" but the test was checking for "November", thus failing. SimpleDateFormat make use of the locale, thus, it fixes the test